phylanx/util/serialization/blaze.hpp

```diff
@@ -0,0 +1,103 @@
+//  Copyright (c) 2017 Parsa Amini
+//
+//  Distributed under the Boost Software License, Version 1.0. (See accompanying
+//  file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+#if !defined(PHYLANX_UTIL_BLAZE_SERIALIZATION_HPP)
+#define PHYLANX_UTIL_BLAZE_SERIALIZATION_HPP
+
+#include <phylanx/config.hpp>
+#include <hpx/include/serialization.hpp>
+#include <blaze/Math.h>
+
+
+namespace hpx { namespace serialization
+{
+    template <typename T>
+    void load(input_archive& ar,
+        blaze::DynamicVector<T, blaze::rowVector>& m,
+        unsigned)
+    {
+        // DeserializeHeader
+        char delineator = ',';
```

Comment:
What is the `delineator` used for? I think it can be removed as there is no need to explicitly separate items in an HPX serialization byte stream.